README.md

```diff
@@ -2,31 +2,241 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain the Java `Object` Class
+- Learn about the various methods that every class inherits
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Every class in Java directly or indirectly inherits Java's `Object` class.
+Consider the following diagram:
 
-## Topic 1
+![Class Hierarchy](https://curriculum-content.s3.amazonaws.com/6677/pillars/object_class_hierarchy.png)
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+In this diagram, the following hierarchy is represented:
 
-## Topic 2
+1. Classes that do not extend any other class explicitly will directly
+   inherit Java's `Object` class:
+    1. `String` extends `Object`.
+    2. `Number` extends `Object`.
+    3. `Person` extends `Object`.
+2. Classes that do extend other classes will then indirectly inherit from
+   Java's `Object` class:
+    1. `Integer` extends `Number`, which extends `Object`.
+    2. `Double` extends `Number`, which extends `Object`.
+    3. `BigDecimal` extends `Number`, which extends `Object`.
+    4. `Student` extends `Person`, which extends `Object`.
+    5. `Teacher` extends `Person`, which extends `Object`.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+_Note: Java's `Object` class is poorly named because an "object" is usually an
```

Comment:
There's an unneeded underscore here, unless this was supposed to be italicized... in which case there's a missing underscore

Reply:
fixed